index.html

```diff
@@ -50,6 +58,203 @@ <h1>Hey there.
                 <img src="./pic/Illustration.png" alt="Illustration">
             </div>
         </section>
+
+
+
+        <section class="works">
+            <div class="title">
+                <h1 id="workstitle">My Recent Works</h1>
+            </div>
+            <div class="frameimg">
+                <div class="imgholder"><img src="./pic/bgc.png" alt="picture"></div>
+                <section class="rightblock">
+                    <h1 id="posttitle">Multi-Post Stories</h1>
```

Comment:
- [ ] I think you used here `h1` as well but for the order of HTML headings consider using `h1` element as a top-level heading only (all “h1” elements are treated as top-level headings by many screen readers and other tools) ao please update it to be `h2` or `h3`.👌🏻